cmd/hypervisor/commands/gen-config.go

```diff
@@ -14,13 +14,15 @@ var (
 	output        string
 	replace       bool
 	configLocType = pathutil.WorkingDirLoc
+	testenv		bool
```

Comment:
Could you please format the file running `gofmt`/`goimports`?

Reply:
Done